Source/Wikitivity.CustomPage/Default.aspx.cs

```diff
@@ -114,32 +121,52 @@ protected void submitJobButton_Click(object sender, EventArgs e)
 
 			try
 			{
-				IRSAPIClient proxy = ConnectionHelper.Helper().GetServicesManager().CreateProxy<IRSAPIClient>(ExecutionIdentity.System);
 				int workspaceID = ConnectionHelper.Helper().GetActiveCaseID();
 				string requestIDGuid = Guid.NewGuid().ToString();
 				int count = 000001;
-
+				//TODO: Can I turn this into a mass op using OM?

```

Comment:
Should and it shouldn't be too much.  You would move this code to a "core" class so that the mass op and the custom page can reference it.